tests/main/gadget-update-pc/task.yaml

```diff
@@ -25,6 +25,14 @@ prepare: |
         exit 1
     fi
 
+    # Set the retain count to 10 to ensure we keep the current gadget snap
+    # intact and can roll back to it. This is easier to get right than to
+    # manually fiddle with re-installing it.
+    snap set core refresh.retain=10
```

Comment:
Why 10 and not 3?

Reply:
I just set it to "something large", I think it's safer than trying to be precise. I'm not strongly attached to it though.

Reply:
Got it. No strong opinion. Sometimes being generous can hide problems. I think this test is precise in what it does (and also is precise below when removing revisions 1000 1001 1002), so I think being precise here would be fine. Anyway... that's a tiny bit.

Reply:
I'll change this to be more precise.

Reply:
Done